services/movie_session.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+from django.db.models.functions import TruncDate
+
+import init_django_orm  # noqa: F401
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie = Movie.objects.get(id=movie_id)
```

Comment:
The `get` method will raise a `DoesNotExist` exception if no `CinemaHall` or `Movie` with the given `id` is found. Consider handling these exceptions to avoid potential runtime errors.

Reply:
We expect the id to be in the database

Reply:
get_or_404

Reply:
Done

Reply:
+